phalcon/config.zep

```diff
@@ -223,6 +270,29 @@ class Config implements \ArrayAccess, \Countable
 	}
 
 	/**
+	 * Sets the default path delimiter
+	 */
+	public static function setPathDelimiter(string! delimiter = null) -> void
```

Comment:
@sergeyklay Static because I thought it would be nice to have the possibility to change the default delimiter for all `Phalcon\Config` instances with one call.